VulnerableApplication/Global.asax.cs

```diff
@@ -17,5 +17,7 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        
+        public string ChangePassword { get; set; }
```

Comment:
[CODE_SMELL] 7 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.